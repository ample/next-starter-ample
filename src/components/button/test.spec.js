import React from 'react';
import renderer from 'react-test-renderer';

import Button from './';

describe('Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Button url="/">testing</Button>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
