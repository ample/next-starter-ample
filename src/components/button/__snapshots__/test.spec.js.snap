// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders correctly 1`] = `
<a
  className="button theme_default"
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
>
  testing
</a>
`;
